### DISCUSSION TITLE
Clear metrics on dashboard

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx#10101] [Q&A] [answered]
How to clear/reset Packet statistics metrics shown on emqx dashboard?

**Comment by hjianbo** [comment, marked as answer]
Hi @alexa22013  It's not supported now.  I'll transfer it to the EMQX repository as a Feature Request

Thanks for your feedback.

**Reply by zmstone** [reply]
@HJianBo  could you link the feature request link here.

**Reply by hjianbo** [reply]
Thanks for the reminder. I have created a Feature Request https://github.com/emqx/emqx/issues/10246

**Comment by zmstone** [comment]
marking this question as marked, we'll move the discussion and status update to the feature request Issue instead.

**Comment by id** [comment]
https://docs.emqx.com/en/emqx/v5.8/admin/api-docs.html#tag/Metrics/paths/~1monitor/delete
